ddtrace/span.py

```diff
@@ -56,6 +56,14 @@
 log = get_logger(__name__)
 
 
+class SpanStore(object):
```

Comment:
might be great to have a docstring on this, I've no clue what it is.

Reply:
Updated, let me know if it makes more sense this way!